### PR TITLE
Add rizin

### PR DESCRIPTION
**Pull request by AbhiTheModder**
This pull request adds support for rizin(fork of radare2), a UNIX-like reverse engineering framework and commandline toolset, to the android termux package. Rizin aims to provide a more user-friendly and modular interface, as well as faster development and bug fixes. With this package, termux users can easily install and use rizin on their android devices, and perform various tasks such as binary analysis, debugging, forensics, and exploitation. The package has been tested on several android versions and architectures and works as expected. I hope this contribution will be useful for the termux community and the rizin project.

and a big thanks to @TomJo2000... I basically took a day of him, but he still helped me. ❤

### CODE REVIEW

**Comment by finagolfin** [comment]
Any reason you are not the maintainer?

**Comment by AbhiTheModder** [comment]
> Any reason you are not the maintainer?

@finagolfin none, if given so I can be the maintainer also. I used @termux  in maintainer as I seen in most of packages and even in example of radare2, that's why.

File: packages/rizin/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Thank you @TomJo2000
```
Comment (Grimler91) [inline packages/rizin/build.sh:1]: Please remove this comment. You can put TomJo as co-developer instead in commit message, like:

```
rizin: add package

Co-developed-by: Joshua Kahn @TomJo2000
```

File: packages/rizin/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+TERMUX_PKG_HOMEPAGE=https://rizin.re/
+TERMUX_PKG_DESCRIPTION="UNIX-like reverse engineering framework and command-line toolset. (fork of radare2)"
+TERMUX_PKG_LICENSE="GPL-3.0, LGPL-3.0"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=0.6.2
+TERMUX_PKG_SRCURL=https://github.com/rizinorg/rizin/archive/v$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SUGGESTS="python, apk-tools, apktool, apksigner"
+TERMUX_PKG_SHA256=ab4bd4c767940b28d683f664f27180665f98bfab2aa7972ba4c331e8bed30da8
```
Comment (Grimler91) [inline packages/rizin/build.sh:9]: Please add newline at end of file

File: packages/rizin/build.sh [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Thank you @TomJo2000
+TERMUX_PKG_HOMEPAGE=https://rizin.re/
+TERMUX_PKG_DESCRIPTION="UNIX-like reverse engineering framework and command-line toolset. (fork of radare2)"
```
Comment (Grimler91) [inline packages/rizin/build.sh:3]: I think we can remove the `(fork of radare2)` comment, the project should hopefully be recognizable on its own

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please rebase and squash the commits against the master branch. 
Right now the PR adds 7 commits, out of which 4 just say `Update build.sh` and `Merge branch 'termux:master' into master`. It should be enough with one commit to add one package

**Comment by AbhiTheModder** [comment]
@Grimler91 Sure thanks and it seems I was directly pushing changes through master branch of my fork that's why I was facing issues during rebasing

Doing it in another branch. Sorry for the troubles

Thanks

**Pull request closed without merging**